keylime-agent/src/revocation.rs

```diff
@@ -498,11 +498,11 @@ pub(crate) async fn worker(
                     cert_absolute_path.display()
                 );
 
-                revocation_cert = match crypto::load_x509(&cert_absolute_path)
-                {
-                    Ok(cert) => Some(cert),
-                    Err(e) => None,
-                };
+                revocation_cert =
+                    match crypto::load_x509_pem(&cert_absolute_path) {
+                        Ok(cert) => Some(cert),
+                        Err(e) => None,
+                    };
```

Comment:
nit: could this be simply `revocation_cert = crypto::load_x509_pem(&cert_absolute_path).ok();`?

Reply:
Thanks, I'll change it